ui/apps/platform/src/services/AuthService/AccessTokenManager.js

```diff
@@ -79,68 +77,23 @@ export default class AccessTokenManager {
         } // nothing to do, operation not started
 
         this.refreshTokenOpPromise = this.options
-            .refreshToken(this.tokenInfo)
-            .then(({ token, info }) => {
-                this.setToken(token, info);
+            .refreshToken()
+            .then(() => {
                 this.refreshTokenOpPromise = null;
             })
             .catch(() => {
                 this.refreshTokenOpPromise = null;
             });
         this.eventEmitter.emit(this.refreshTokenSymbol, this.refreshTokenOpPromise);
+        // Do a refresh every hour. For now this is statically defined, since we do not have access to the
+        // underlying token and its expiration time.
+        const expiryDate = subSeconds(new Date(), 3600);
+        const delay = differenceInMilliSeconds(expiryDate, Date.now());
+        this.refreshTimeout.set(this.refreshToken, delay);
```

Comment:
This probably does not make much sense since it defeats the purpose of the early refresh. Let's either remove it altogether or store the access token expiration in local storage so we can maintain the current early refresh logic.

Reply:
I've removed the periodic refresh logic now completely for the time being, let's see whether we have strong opinions on keeping it intact or can drop the whole "store something in local storage" completely.